pkg/api/vlabs/validate_test.go

```diff
@@ -35,7 +35,7 @@ func Test_OrchestratorProfile_Validate(t *testing.T) {
 
 func Test_KubernetesConfig_Validate(t *testing.T) {
 	// Tests that should pass across all versions
-	for _, k8sVersion := range []string{Kubernetes153, Kubernetes157, Kubernetes160, Kubernetes162, Kubernetes166, Kubernetes170, Kubernetes171, Kubernetes172} {
+	for _, k8sVersion := range []string{Kubernetes153, Kubernetes157, Kubernetes160, Kubernetes162, Kubernetes166, Kubernetes168, Kubernetes170, Kubernetes171, Kubernetes172} {
```

Comment:
Because the versions to test are defined in the test itself instead of in the code, the tests incorrectly passed when I forgot to add `Kubernetes168` to the list of allowed versions. We need to fix this test before merging.